src/SprykerSdk/Spryk/Model/Spryk/Definition/Argument/Callback/GlueProcessorModelTargetFilenameCallback.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Callback;
+
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface;
+
+class GlueProcessorModelTargetFilenameCallback implements CallbackInterface
+{
+    protected const CALLBACK_NAME = 'GlueProcessorModelTargetFilename';
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::CALLBACK_NAME;
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface $argumentCollection
+     * @param mixed|null $value
+     *
+     * @return mixed
+     */
+    public function getValue(ArgumentCollectionInterface $argumentCollection, $value)
+    {
+        return sprintf(
```

Comment:
Please check if it's already there.

Reply:
This method works with className argument, which shouldn't contain `.php` affix, so I don't think that we need additional check here.